builtin/json.mbt

```diff
@@ -21,3 +21,46 @@ pub enum Json {
   Array(Array[Json])
   Object(Map[String, Json])
 } derive(Eq)
+
+/// Trait for types that can be converted to `Json`
+pub trait ToJson {
+  to_json(Self) -> Json
+}
+
+pub fn Bool::to_json(self : Bool) -> Json {
+  if self {
+    True
```

Comment:
I am thinking of we should rename True into JsonTrue to avoid name pollution, note JsonTrue is used much less after we introduce first class json support